### ISSUE TITLE
Running OSRM instance not responding to any request.

### ISSUE THREAD

**Issue by amit1rrr** [Project-OSRM/osrm-backend#4276]
I am running osrm-routed on a planet extract. Server starts up fine (image attached) but when I make any requests using curl there is no response indefinitely. With netstat output I can see that the process is running at the specified port. Instance has enough free memory (image attached). 

Seems like somehow osrm is not starting up fully. Requests don't even reach the running osrm instance. Any ideas?

![screen shot 2017-07-11 at 6 41 29 pm](https://user-images.githubusercontent.com/1350490/28069866-9519f9cc-6668-11e7-86e3-f3694e7df836.png)
![screen shot 2017-07-11 at 6 47 22 pm](https://user-images.githubusercontent.com/1350490/28070089-6640f604-6669-11e7-9da4-1de7e7994878.png)

**Comment by MoKob** [comment]
There seems to be indeed something going on with starting your server. The full output should look something like:

```
[info] starting up engines, v5.9.0
[info] Threads: 8
[info] IP address: 0.0.0.0
[info] IP port: 5000
[info] load names from: "data/map.osrm.names"
[info] set checksum: 3143603039 (5.9 addition (?))
[info] http 1.1 compression handled by zlib version 1.2.8
[info] Listening on: 0.0.0.0:5000
[info] running and waiting for requests
```

The part about `running and waiting for requests` is missing from your output.
How long have you waited between starting and serving requests? Loading the data takes quite a while, which isn't shown when starting osrm-routed.
So it might be the server is still loading.
To ensure data is loaded, you could try first loading the data into shared memory:

```
osrm-datastore data/map.osrm
[info] Loading data into REGION_1
[info] load names from: "data/map.osrm.names"
[info] Allocating shared memory of 72580 bytes
[info] All data loaded. Notify all client about new data in REGION_1 with timestamp 1095
[info] Marking old shared memory region REGION_2 for removal... 
[info] ok.
[info] Waiting for clients to detach... 
[info]  ok.
[info] All clients switched.
```

When this is finished, you can start-up OSRM with `osrm-routed -s` and the startup should be almost instant.

If that doesn't work, could you try and start up the server in debug mode and see if any assertions are thrown?

**Comment by amit1rrr** [comment]
It started up properly after waiting for a long time. I don't know exactly how long it took but I saw after few hours and server was up and running with last line as `running and waiting for requests`. 

I also tried the osrm-datastore option and that was much quicker. That loaded the data in shared memory quickly and following `osrm-routed -s` was quick as you said.

For other folks reading this: bottom line is server is not really up unless you see `running and waiting for requests`. This could take long time for large extracts.

I'm not sure about the sequence of actions. But my recommendation would be to load all necessary data first and then start listening on the port to avoid this kind of confusion. Resolving this issue as my problem is solved. Thanks for the support.
